OpenIPSL/Electrical/Renewables/PSSE/AddOnBlocks.mo

```diff
@@ -3,6 +3,7 @@ package AddOnBlocks
   "This package contains additional add ons that can be added to the original renewable models."
   model IrradianceToPower "PV Array Power Output from Irradiance."
 
+    parameter Modelica.Units.SI.ApparentPower M_b = 100000000;
```

Comment:
You are missing the description string. What is this parameter supposed to represent.

Reply:
This was fixed.